src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IFixedStatement.cs

```diff
@@ -35,18 +35,21 @@ void M1()
             string expectedOperationTree = @"
 IOperation:  (OperationKind.None, Type: null) (Syntax: 'fixed(int * ... }')
   Children(2):
-      IVariableDeclarationsOperation (1 declarations) (OperationKind.VariableDeclarations, Type: null) (Syntax: 'int *p = &i')
-        IVariableDeclarationOperation (1 variables) (OperationKind.VariableDeclaration, Type: null) (Syntax: 'p = &i')
-          Variables: Local_1: System.Int32* p
+      IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'int *p = &i')
```

Comment:
>IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null) (Syntax: 'int *p = &i') [](start = 6, length = 128)

I think we shouldn't have this node in the tree for this scenario. Syntactically we have VariableDeclarationSyntax, which maps to IVariableDeclarationOperation